ros2bag/ros2bag/verb/record.py

```diff
@@ -60,6 +60,12 @@ def add_arguments(self, parser, cli_name):  # noqa: D102
                   'Default it is zero, recording written in single bagfile and splitting '
                   'is disabled.'
         )
+        parser.add_argment(
+            '-d', '--max-bag-duration', type=int, default=0,
+            help='maximum duration in nanosecnds before the bagfile will be split. '
```

Comment:
```suggestion
            help='maximum duration in nanoseconds before the bagfile will be split. '
```

We probably want to specify what happens when both splitting arguments are provided - I'm guessing it's something like "will be split at the duration or the size, whichever comes first"

Also nanoseconds are a bit unwieldy since likely users are thinking in minutes for bagfile durations, so they'll have to type at least 10 digits. I think that seconds would be a better atomic unit for splitting.

Reply:
I started in nanoseconds since that's what the internal times were set in. But if people are thinking in minutes, then I completely agree seconds would be a better unit of time. I'll set to changing that now.